content/docs/app-developer-guide/environment-variables.md

```diff
@@ -91,4 +85,41 @@ The following environment variables were set and available to buildpacks at buil
 
 > **NOTE:** Variables defined using `--env` take precedence over variables defined in `--env-file`.
 
+### Using Project Descriptor
+The `--descriptor` parameter must be a path to a file which follows the project.toml [schema][descriptor-schema].
+You can define environment variables in an `env` table in the file, and pass those into the application.
+
+##### Example:
+```bash
+# clone the repo
+git clone https://github.com/buildpacks/samples
+
+# Add an environment variable to the project.toml
+printf "[[build.env]]\nname='HELLO'\nvalue='WORLD'" >> samples/apps/bash-script/project.toml
```

Comment:
This may be a matter of preference and worth discussing but I personally find that here docs are easier to read for multiline writing.

```suggestion
cat >> samples/apps/bash-script/project.toml <<EOL
[[build.env]]
name="HELLO"
value="WORLD"
EOL
```